requirements.txt

```diff
@@ -6,7 +6,7 @@ cssselect>=1.1.0,<1.2
 html5lib>=1.1,<1.2
 isodate>=0.6.0
 json_home_client>=1.1.1,<2
-lxml>=4.5,<4.6
+lxml>=4.5
```

Comment:
I wasn't sure if this is going to cause any issues with the Pipfile.lock, but it looks like dependabot also updated it now to 4.6.2, so maybe it should be >=4.6 now